region/brinstar/red/Red Brinstar Fireflea Room.json

```diff
@@ -557,6 +518,133 @@
       "requires": [
         "h_canCrystalFlash"
       ]
+    },
+    {
+      "link": [2, 3],
```

Comment:
[2,3] could use a damageboost strat since it's not as hard as the full room waver ride.  But it would still be notable and not that much easier, so maybe don't bother 🤷